call-for-participation.html

```diff
@@ -326,7 +326,7 @@ <h2 id="sponsorship">How can my organization sponsor the workshop series?</h2>
 </p>
 
 
-<h2 id="programming-committee">Program Committee</h2>
+<h2 id="program-committee">Program Committee</h2>
```

Comment:
```suggestion
<h2 id="program-committee"><a id="programming-committee"></a>Program Committee</h2>
```

(To avoid breaking any links.)